app/questionnaire/rules.py

```diff
@@ -282,17 +282,18 @@ def evaluate_when_rules(when_rules, schema, metadata, answer_store, group_instan
     """
 
     for when_rule in when_rules:
+        group_instance_something = group_instance
```

Comment:
group_instance_something?  The name isn't very expressive I think.

Reply:
haha, yeh I forgot about that. I was trying it to see if it fixed my issue (which it did) then forgot to call it something sensible